apps/api/src/server/utils/mapping/map-zoom-level.js

```diff
@@ -3,8 +3,15 @@ import { pick } from 'lodash-es';
 /**
  *
  * @param {import('@pins/api').Schema.ZoomLevel} zoomLevel
- * @returns {{name: string, displayNameEn: string, displayNameCy: string}}
+ * @returns {{name: string, displayOrder: number, displayNameEn: string, displayNameCy: string}}
  */
 export const mapZoomLevel = (zoomLevel) => {
-	return pick(zoomLevel, ['id', 'name', 'displayNameEn', 'displayNameCy']);
+	return pick(zoomLevel, [
+		'id',
+		'name',
+		'displayOrder',
+		'displayNameEn',
+		'displayNameCy',
+		'displayName'
```

Comment:
Do we have a `displayName` as well?

Reply:
no,,, we have id, name, displayOrder, displayNameEn, DisplayNameCY... let me correct that